src/test/java/net/minecraftforge/debug/gameplay/level/ForcedChunkLoadingTest.java

```diff
@@ -0,0 +1,42 @@
+package net.minecraftforge.debug.gameplay.level;
```

Comment:
needs license header, and the mod id doesnt need the _test suffix
You could also pick a spot that you KNOW is outside the range by checking the chunkmap. Or offsetting the position from the current test position.

Also, need to add a check to make sure the chunk is loaded and in the ticking list.